### ISSUE TITLE
Add list generators command / move current commands under "generate"

### ISSUE THREAD

**Issue by ovflowd** [nodejs/api-docs-tooling#148]
### Enter your suggestions in details:

- There should be a simple top-level command called `list` to list the current generators. (Version, Name, Description, Examples)
- The current top-level command to generate API docs should be called "generate"
- Help should also be a "top" level command and be accessible within each subcommand.

cc @canerakdas @RedYetiDev for opinions

**Comment by RedYetiDev** [comment]
I agree with the idea of listing the generators, but I don't know if it needs to be "list". It could just be "generate" without any arguments.

Regardless on that slight difference or not, +1

**Comment by canerakdas** [comment]
I think it's a great idea 🙇 As far as I understand, we will be able to use the existing mandatory fields with top-level `generate` and other commands (`help`, `list`, etc.) at the top-level

Bonus:
We can add a top-level wizard, ask questions, and able to run interactive generators like [sentry-wizard](https://github.com/getsentry/sentry-wizard)

A tool that we will use internally may seem like overkill, but IMO it will make things a lot easier for those who want to contribute or try it for the first time

**Comment by avivkeller** [comment]
See https://github.com/nodejs/api-docs-tooling/pull/250
